### ISSUE TITLE
[CI] migration and starter zips contain buildscript with not updated version

### ISSUE THREAD

**Issue by TheElan** [SinTh0r4s/ExampleMod1.7.10#18]
Update happens on next automatic commit and CI is skipped, maybe we shouldn't skip CI and just make it so only this workflow is triggered on it

**Comment by SinTh0r4s** [comment]
Suboptimal, but it will propose an update. Wouldn't give it too much proprity, since it is only an inconvenience and solved with a move to a plugin.

**Comment by TheElan** [comment]
yeah, maybe at that point plugin is much better solution to version it. It doesn't look like plugin writing is too different from writing buildscript, you just need to learn the API, maybe we should make it as a part of FG support for 7x gradle
